e2e/accessibility_test/accessibility.spec.js

```diff
@@ -0,0 +1,190 @@
+const { setup, screen } = require('../../src/index');
+
+describe('accessibility', () => {
+    beforeAll(async () => {
+        await setup.LaunchLoop();
+    });
+    describe('home', () => {
+        it('has a Active Carbohydrates Label', async () => {
```

Comment:
[nit] Is there anyway to shorten this syntax? Not sure. It is repeated a lot.
```
async () => {
  await expect(<something>).toExist();
}
```
This might be the shortest possible form given Javascript, but thought I'd ask if there was anything shorter that still made sense (and wasn't cryptic).

Reply:
yeah true - I hadn't thought of that